pytest_tests/helpers/grpc_responses.py

```diff
@@ -7,6 +7,7 @@
 # Regex patterns of status codes of Object service (https://github.com/nspcc-dev/neofs-spec/blob/98b154848116223e486ce8b43eaa35fec08b4a99/20-api-v2/object.md)
 MALFORMED_REQUEST = "code = 1024.*message = malformed request"
 WRONG_CONTAINER = "code = 1024.*message = wrong container"
```

Comment:
now unused?

Reply:
I'd leave it, it can be useful for new tests.

Reply:
dont mind